internal/service/cluster/api/openapi.yaml

```diff
@@ -0,0 +1,826 @@
+#
+# Copyright (c) 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and limitations under the
+# License.
+#
+
+openapi: "3.0.0"
```

Comment:
skipping reviewing this since the standard is not yet provided

Reply:
The draft is here if you feel inclined to have a look:  [here](https://api.media.atlassian.com/file/f4e983b4-31d9-4892-ad04-cc60fad1d5fc/binary?token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiIwOTA4N2MzYi0xNGU1LTQzM2YtOTZhYy03MjIyZjFkN2RiYTgiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOmY0ZTk4M2I0LTMxZDktNDg5Mi1hZDA0LWNjNjBmYWQxZDVmYyI6WyJyZWFkIl19LCJleHAiOjE3MzQ0NTExMDksIm5iZiI6MTczNDM2ODE4OX0.pSLG9-fQrTCVDyURJjCeWL5VlsLc2hp0vj_JN27U4kk&client=09087c3b-14e5-433f-96ac-7222f1d7dba8&name=O-RAN.WG6.TS.O-CLOUD-IM.0-R004-v03.00.pdf&dl=true)

Reply:
can't see it, but no need anyway